app/controllers/MarriageAllowanceController.scala

```diff
@@ -44,21 +44,20 @@ class MarriageAllowanceController @Inject()(marriageAllowanceService: MarriageAl
             user_record = Some(recipientRecord),
             availableYears = Some(taxYears),
             status = ResponseStatus(status_code = "OK"))))
-        case Left(_: DataRetrievalError) =>
-          NotFound(Json.toJson(GetRelationshipResponse(
+        case Left(_: DataRetrievalError) => Ok(Json.toJson(GetRelationshipResponse(
           status = ResponseStatus(status_code = RECIPIENT_NOT_FOUND))))
       } recover {
         case error: ServiceError =>
           logger.warn(error.getMessage)
-          NotFound(Json.toJson(GetRelationshipResponse(
```

Comment:
@IanPGodman how have you managed to do all the changes in this file?

Reply:
I thought I had restored all Milo changes which seemed to have been reverted. I will check again to make sure I got them all.

Reply:
I would like to know how this happened so we can understand and ensure it doesn't happen again.